### PR TITLE
add documentation

### PR DESCRIPTION
**Pull request by caedstrom**
added link to our box standard documentation

### CODE REVIEW

**Review by cherrypj — CHANGES_REQUESTED** [review]
Fix the typo @caedstrom

File: services/drupal/web/themes/epa_theme/source/_patterns/05-components/box/box.twig [hunk 1]
```diff
@@ -51,3 +51,11 @@
     </div>
   {% endif %}
 </div>
+
+{% if is_demo %}
+  <div class="usa-section usa-section--light margin-y-3">
+    <div class="l-constrain">
+      <p><strong>Related link</strong>: <a href=”https://www.epa.gov/web-policies-and-procedures/web-standard-boxes”>Web Standard: Boxes</a></p>
```
Comment (dcmouyard) [inline services/drupal/web/themes/epa_theme/source/_patterns/05-components/box/box.twig]: Be careful with `”` and `”` characters: they should be `"` here in the code.

File: services/drupal/web/themes/epa_theme/source/_patterns/05-components/box/box.twig [hunk 2]
```diff
@@ -51,3 +51,11 @@
     </div>
   {% endif %}
 </div>
+
+{% if is_demo %}
```
Comment (dcmouyard) [inline services/drupal/web/themes/epa_theme/source/_patterns/05-components/box/box.twig]: Moving forward, the `is_demo` variable should be replaced with the `show_documentation` variable when wrapping documentation.

**Review by dcmouyard — APPROVED** [review]
I pointed two things to keep in mind for future PRs. I’ve fixed these issues in my latest commit.

**Pull request merged**